src/TTextEdit.cpp

```diff
@@ -862,6 +863,29 @@ void TTextEdit::normaliseSelection()
     }
 }
 
+void TTextEdit::expandSelectionToWords() {
+    int yind = mPA.y();
+    int xind = mPA.x();
+    for (; xind >= 0; --xind) {
+        if (mpBuffer->lineBuffer.at(yind).at(xind) == QChar::Space
+            || mpHost->mDoubleClickIgnore.contains(mpBuffer->lineBuffer[yind].at(xind))) {
```

Comment:
```cpp
            || mpHost->mDoubleClickIgnore.contains(mpBuffer->lineBuffer.at(yind).at(xind))) {
```
ℹ️ As one does not need to modify the line (a `QString`) of text in `lineBuffer` (a `QStringList`) it is faster/better to use `(const QString&) QStringList::at(const int)` rather than `(QString&) QStringList::operator[](const int)` here...! 😁